ckan/templates/ckan-pvc.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
 spec:
   storageClassName: {{ .Values.ckanStorageClassName }}
   accessModes:
-    - ReadWriteMany
+    - {{ .Values.diskAccessMode | default "ReadWriteMany" }} 
```

Comment:
let's rename this to ckanStorageDiskAccessMode

Reply:
@akariv fixed